Products/MimetypesRegistry/tests/test_mimetypes.py

```diff
@@ -1,16 +1,21 @@
 # -*- coding: utf-8 -*-
-from .utils import input_file_path
-from plone.app.testing.bbb import PloneTestCase as ATSiteTestCase
+from Products.MimetypesRegistry.tests.utils import input_file_path
+# from plone.app.testing.bbb import PloneTestCase as ATSiteTestCase
```

Comment:
This could go, right?

This and the changelog entry and I will merge it :+1:

Reply:
there is more work pending on the test in py3